src/lib/variable.js

```diff
@@ -0,0 +1,42 @@
+// datastream grouped by variableCode
+export async function getvariables({ helpers, params, variableCodes }) {
```

Comment:
Fix capitalization. `getVariables`

Reply:
Do we need the `variableCodes` Set for this? Can we just use the existing `variables` Map?